daemon/src/test/java/org/sonarlint/daemon/services/StandaloneSonarLintImplTest.java

```diff
@@ -19,130 +19,29 @@
  */
 package org.sonarlint.daemon.services;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.stream.Collectors;
+import java.util.Arrays;
 
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
-import org.mockito.ArgumentCaptor;
-import org.sonarlint.daemon.services.StandaloneSonarLintImpl.DefaultClientInputFile;
-import org.sonarlint.daemon.services.StandaloneSonarLintImpl.ProxyIssueListener;
-import org.sonarlint.daemon.services.StandaloneSonarLintImpl.ProxyLogOutput;
-import org.sonarsource.sonarlint.core.client.api.common.LogOutput.Level;
-import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Issue;
-import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Issue.Severity;
-import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.LogEvent;
 import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.StandaloneConfiguration;
 
-import io.grpc.StatusRuntimeException;
 import io.grpc.stub.StreamObserver;
 
 public class StandaloneSonarLintImplTest {
   @Rule
   public TemporaryFolder temp = new TemporaryFolder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2924)